README.md

```diff
@@ -1,4 +1,4 @@
-<img src="Thanos-remote-read_SignatureLogo_RGB-Black.svg" width="180" alt="thanos-remote-read">
+<img src="https://github.com/G-Research/thanos-remote-read/blob/logo-folder/docs/logo/png/Thanos-remote-read_SignatureLogo_RGB-Black.png" width="180" alt="thanos-remote-read">
```

Comment:
I think we want the source to be relative instead, and we want to use the vector version. The suggestion below _should_ work on both your repo and this one.

```suggestion
<img src="docs/logo/svg/Thanos-remote-read_SignatureLogo_RGB-Black.svg" width="180" alt="thanos-remote-read">
```

Reply:
Doy! Yes, you are right. I've committed your suggestion now, thanks😊